src/rgw/rgw_rados.cc

```diff
@@ -3309,7 +3309,7 @@ int RGWRados::get_required_alignment(const rgw_pool& pool, uint64_t *alignment)
     return r;
   }
 
-  bool requires;
+  bool requires = false;
```

Comment:
@agutikov should be initialized by the following call, if it doesn't then we need to fix librados